src/app/meta-campaign/meta-campaign.component.spec.ts

```diff
@@ -55,6 +55,7 @@ describe('MetaCampaignComponent', () => {
       'Test Master Campaign!',
       [],
       [],
+      '',
```

Comment:
@tbg-jasonfung surely the optional end param should be omitted if we're trying to simulate what happens for meta-campaigns where it's not set? This is why the param is now optional!

Reply:
@NoelLH Yup you're right, I went a bit update trigger happy